iceoryx2-bb/posix/src/read_write_mutex.rs

```diff
@@ -411,11 +343,6 @@ impl<'a, T: Sized + Debug> ReadWriteMutex<'a, T> {
         Self { handle }
     }
 
-    /// Returns the used clock type of the mutex
-    pub fn clock_type(&self) -> ClockType {
-        unsafe { *self.handle.clock_type.get() }
-    }
-
     /// Blocks until a read-lock could be acquired and returns a [`MutexReadGuard`] to provide
     /// read access to the underlying value.
     pub fn read_lock(&self) -> Result<MutexReadGuard<'_, '_, T>, ReadWriteMutexReadLockError> {
```

Comment:
Can you add the prefix `blocking` -> `blocking_read_lock()` so that it is clear for everyone that this is a blocking function. Can you do the same thing for the write lock and the lock functions in `Mutex`.
In the end, it would be nice to always have the 3 methods. `try_`, `timed_` and `blocking_` whatever.